files-community/Files
# Feature: Added support for resizing columns in the column layout

**Resolved / Related Issues**

Closes #6534

**Validation**
How did you test these changes?

    1. Open app
    2. Enter Desktop for example
    3. Choose Columns mode
    4. Try resizing the columns by holding the border and dragging it horizontally

**Screenshots (optional)**
